.github/workflows/java.yml

```diff
@@ -64,6 +64,12 @@ jobs:
                   target: ${{ matrix.os == 'ubuntu-latest' && 'x86_64-unknown-linux-gnu' || 'x86_64-apple-darwin' }}
                   github-token: ${{ secrets.GITHUB_TOKEN }}
 
+            - name: Install protoc (protobuf)
```

Comment:
Should be in https://github.com/Bit-Quill/glide-for-redis/blob/3158103dc7f599a3af35bf52bf043c1631192188/.github/workflows/install-shared-dependencies/action.yml#L56-L60